packages/cli/src/constants.ts

```diff
@@ -10,3 +10,8 @@ export const ROOT_API_URL_PROD = 'https://api.subquery.network';
 export const BASE_PROJECT_URL = 'https://project.subquery.network';
 
 export const BASE_TEMPLATE_URl = 'https://templates.subquery.network';
+
+// Regex for cold tsManifest
+// export const ENDPOINT_REG = /endpoint:\s*(?:\[)?\s*['"]([\s\S]+?)['"]\s*(?:\])?/;
+export const ENDPOINT_REG = /endpoint:\s*((?:\[\s*['"][^'"]+['"]\s*(?:,\s*['"][^'"]+['"]\s*)*\])|['"][^'"]+['"])/;
```

Comment:
```suggestion
export const ENDPOINT_REG = /endpoint:\s*((?:\[\s*("|'|`)(\S*)\1(?:,\s*(("|'|`)(\S*)\1)\s*)*\])|("|'|`)(\S*)\1);
```

I've improved this a little, it adds support for back ticks and checks that the opening quote character matches the closing one https://regexr.com/7ldti